apps/admin/src/pages/Projects/Projects.tsx

```diff
@@ -0,0 +1,205 @@
+
+import GithubIcon from '@/public/assets/github.svg';
+import ExternalLink from '@/public/assets/external-link.svg';
+import { Button, Modal, LoaderOverlay } from 'ui';
+
+import {
+  useDeleteProjectsMutation,
+  useGetProjectsAdminQuery,
+} from 'apollo-hooks';
+
+function Index() {
+  const [openDeleteModal, setOpenDeleteModal] = useState(false);
+
+  const { data, loading } = useGetProjectsAdminQuery();
+
+  const [deleteProject, { loading: deleteProjectLoading }] =
+    useDeleteProjectsMutation();
+
+  const deleteProjectClick = async (projectId) => {
+    setOpenDeleteModal(false);
+    try {
+      const deleteData = await deleteProject({
+        variables: {
+          projectIds: [projectId],
+        },
+      });
+      if (deleteData?.data?.deleteProjects?.length > 0) {
+        toast.success('Project deleted successfully');
+      }
+    } catch (error) {
+      toast.error('Project deleted failed');
+    }
+  };
+
+  const columns = useMemo(
+    () => [
+      {
+        accessorKey: 'title',
+        header: () => <span>Title</span>,
+        size: 300,
+        cell: (info) => {
+          return (
+            <div className='flex flex-row gap-[10px] items-center'>
+              <div className='flex max-h-[70px] max-w-[70px]'>
+                <Image
+                  loader={() => info?.row?.original?.preview}
+                  src={info?.row?.original?.preview}
+                  alt={info?.row?.original?.preview}
+                  width={100}
+                  height={100}
+                  style={{
+                    borderRadius: '50%',
+                  }}
+                />
+              </div>
+
+              <div className='flex flex-col gap-[5px]'>
+                <span className='w-full text-gray-700 font-medium text-[16px]'>
+                  {info?.getValue()}
+                </span>
+
+                <div className='flex gap-[5px] flex-wrap'>
+                  {info?.row?.original?.tags?.map((tag, i) => (
+                    <span
+                      key={i}
+                      className='capitalize w-fit flex items-center bg-[#e5e7eb] px-[10px] py-[2px] rounded-lg text-gray-700 font-medium text-[14px]'
+                    >
+                      {tag}
+                    </span>
+                  ))}
+                </div>
+              </div>
+            </div>
+          );
+        },
+      },
+      {
+        accessorKey: 'author',
+        width: 200,
+        header: () => <span>Author</span>,
+        cell: (info) => {
+          return (
+            <div className='flex flex-row gap-[10px] items-center'>
+              <div className='flex max-h-[70px] max-w-[70px]'>
+                <Image
+                  loader={() => info?.row?.original?.author?.avatar}
+                  src={info?.row?.original?.author?.avatar}
+                  alt={info?.row?.original?.author?.avatar}
+                  width={'100%'}
+                  height={'100%'}
```

Comment:
```suggestion
                  width="100%"
                  height="100%"
```
are you sure this works? I think you need to another prop to have dynamic width and heigth